lib/Fencer/Types.hs

```diff
@@ -188,4 +188,4 @@ data RuleBranch = RuleBranch
 ----------------------------------------------------------------------------
 
 -- | A network port wrapper
-newtype Port = Port Word deriving newtype (Eq, Show)
+newtype Port = Port { getPort :: Word } deriving newtype (Eq, Show)
```

Comment:
There are two conventions on how to prefix newtype unwrappers: with `get` like you did (that's what `base` does) and with `un` like people often do in production. For me `getPort` means a function that somehow gets, you know, a `Port`, so I like the `un` convention. Just saying, it's not important.

Reply:
Thanks for the note! I adopted your proposal with `un`.